### PR TITLE
Form group responsive style

### PR DESCRIPTION
**Pull request by Th3M3**
**By submitting this pull request, I confirm the following (please check boxes, eg [X] - no spaces) _Failure to fill the template will close your PR_:**

***Please submit all pull requests against the `development` branch. Failure to do so will delay or deny your request***

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x] I have checked that [another pull request](https://github.com/pi-hole/pi-hole/pulls) for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**How familiar are you with the codebase?:**

_3_

---
I have found out that the input field in `queryads.php` is really small if it gets viewed on a smartphone.
I'm sure the same issue have been solved in the past for the black/whitelist page.
So here is my - a litte bit more elegant - version.

Maybe the comment line can be better formulated. My english is not the best.
And I am free to change the width threshold. Actually I have set it to `991`px.

**Before** (on smartphone - portrait):

**After** (on smartphone - portrait):

**After** (on smartphone - landscape):

**On large screen it looks like before**:
pihole_fullscreen

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

### CODE REVIEW

**Comment by PromoFaux** [comment]
Do you think you could split the two buttons, so that they appear central below the input box? I'm guessing with 'col-xs-5' and 'col-xs-offset-2 col-xs-5' respectively. Not near a computer to try this at the moment, but I think that should work...

**Comment by Th3M3** [comment]
@PromoFaux 
I think what you want can be done by changing the function to this:
<pre>
$(window).on("resize",function() {
    if ($(window).width() < 991) {
        $(".form-group.input-group").removeClass("input-group").addClass("input-group-block");
        $(".form-group.input-group-block > input").css("margin-bottom", "5px");
        $(".form-group.input-group-block > .input-group-btn").removeClass("input-group-btn").addClass("btn-block text-center");
    }
    else {
        $(".form-group.input-group-block").removeClass("input-group-block").addClass( "input-group" );
        $(".form-group.input-group > input").css("margin-bottom","");
        $(".form-group.input-group > .btn-block.text-center").removeClass("btn-block text-center").addClass("input-group-btn");
    }
});
$(document).ready(function() {
    $(window).trigger("resize");
});
</pre>
So it would look like this:

**Comment by PromoFaux** [comment]
That looks good. It's actually making me wonder why we have additional javascript to handle screen resizing, when bootstrap has the `col-xs/md/lg` built into it...

**Comment by Th3M3** [comment]
I've tried different ways and also adding classes `col-xs/md/lg`, but nothig works as expected and like it does with js.
Maybe someone can find a way to do this with bootstrap.

File: scripts/pi-hole/js/queryads.js [hunk 1]
```diff
@@ -122,3 +122,21 @@ $("#btnSearchExact").on("click", function() {
     exact = "exact";
     eventsource();
 });
+
+// Wrap form-group's buttons to next line when viewed on a small screen
+// Wrap form-group's buttons to next line when viewed on a small screen
```
Comment (Mcat12) [inline scripts/pi-hole/js/queryads.js]: Double comments

**Comment by PromoFaux** [comment]
I'll try and have more of a play after work. I kind of got there with it last night, but not quite!

Good work so far, though!

**Comment by DL6ER** [comment]
Approved

**Comment by Th3M3** [comment]
Ok, done.
And I updated the code to center and spearate the buttons, like @PromoFaux requested it.

**Comment by DL6ER** [comment]
Approved

**Comment by Mcat12** [comment]
Approved

**Pull request merged**